native/general_double.go

```diff
@@ -10,6 +10,21 @@ import (
 	"math"
 )
 
+func checkMatrix(m, n int, a []float64, lda int) {
```

Comment:
Ohhh, that was just showing up as an executable, and I didn’t realize it was just a text file! Oops
> On Apr 28, 2016, at 2:56 PM, Dan Kortschak <notifications@github.com> wrote:
> 
> In native/general_double.go <https://github.com/gonum/blas/pull/177#discussion_r61500573>:
> 
> > @@ -10,6 +10,21 @@ import (
> >  	"math"
> >  )
> >  
> > +func checkMatrix(m, n int, a []float64, lda int) {
> https://github.com/gonum/blas/blob/master/native/single_precision <https://github.com/gonum/blas/blob/master/native/single_precision>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/gonum/blas/pull/177/files/f22d4fe0e2f9e5f0ec7d6b85e42685b77ee193cc#r61500573>